### PR TITLE
privacy page ok

### PR DESCRIPTION
**Pull request by Mussuranae**
Privacy page. Last page of issue #1170

### CODE REVIEW

File: apps/catalog/marketplace/marketplace/src/app/pages/privacy-page/privacy-page.component.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  <h1>Privacy</h1>
+</header>
+
+<section fxLayout="row" fxLayoutAlign="center">
+  <mat-card>
+    <mat-card-content>
+      <article>
+        <h3>Privacy Policy</h3>
+        <p>
+          XXXXXX is a brand from the company Cascade 8.
```
Comment (Hugo-Hontans) [inline apps/catalog/marketplace/marketplace/src/app/pages/privacy-page/privacy-page.component.html]: Care you write `XXXXXX`

File: apps/catalog/marketplace/marketplace/src/app/pages/privacy-page/privacy-page.component.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+          Cascade 8 also collects potentially personally-identifying information like Internet Protocol (IP) addresses for logged in users and for users leaving comments on http://Cascade 8.com blog posts. Cascade 8 only discloses logged in user and commenter IP addresses under the same circumstances that it uses and discloses personally-identifying information as described below.
+        </p>
+      </article>
+
+      <article>
+        <h3>Gathering of Personally-Identifying Information</h3>
+        <p>
+          Certain visitors to Cascade 8's websites choose to interact with Cascade 8 in ways that require Cascade 8 to gather personally-identifying information. The amount and type of information that Cascade 8 gathers depends on the nature of the interaction. For example, we ask visitors who sign up for a blog at http://Cascade 8.com to provide a username and email address.
```
Comment (Hugo-Hontans) [inline apps/catalog/marketplace/marketplace/src/app/pages/privacy-page/privacy-page.component.html]: `Certain`

File: apps/catalog/marketplace/marketplace/src/app/pages/privacy-page/privacy-page.component.html (hunk 1 above)
Comment (Mussuranae) [reply apps/catalog/marketplace/marketplace/src/app/pages/privacy-page/privacy-page.component.html]: Yeah, that's a very good name, no ? :D

File: apps/catalog/marketplace/marketplace/src/app/pages/privacy-page/privacy-page.component.html (hunk 2 above)
Comment (Mussuranae) [reply apps/catalog/marketplace/marketplace/src/app/pages/privacy-page/privacy-page.component.html]: Text will change soon :)

**Pull request merged**